src/Symfony/Component/Notifier/Bridge/Slack/Block/SlackHeaderBlock.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Slack\Block;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+
+/**
+ * @author Tomas Norkūnas <norkunas.tom@gmail.com>
+ */
+final class SlackHeaderBlock extends AbstractSlackBlock
+{
+    private const TEXT_LIMIT = 150;
+    private const ID_LIMIT = 255;
+
+    public function __construct(string $text)
+    {
+        if (\strlen($text) > self::TEXT_LIMIT) {
```

Comment:
Does slack count in bytes or in utf8?
If utf8, then this is too restrictive.
Note that the same question exists on the Discord notifier, if someone would like to have a look at it.

Reply:
`str_repeat('स्ते', 150)`

`{"ok":false,"error":"invalid_blocks","warning":"missing_charset","response_metadata":{"messages":["[ERROR] failed to match all allowed schemas [json-pointer:\/blocks\/0\/text]","[ERROR] must be less than 151 characters [json-pointer:\/blocks\/0\/text\/text]"],"warnings":["missing_charset"]}}`